cuesubmit/cuesubmit/Validators.py

```diff
@@ -25,7 +25,7 @@ def matchLettersAndNumbersOnly(value):
 
 def matchNoSpecialCharactersOnly(value):
     """Match strings containing letters, numbers, '-', and '_'"""
-    if re.match('^[a-zA-Z0-9-_ ]+$', value):
+    if re.match('^[a-zA-Z0-9.-_ ]+$', value):
```

Comment:
Does `.` need an escape character?

Reply:
Since we're in a character class `.` does not need to be escaped. However, the `-` should be.